README.md

```diff
@@ -1,2 +1,9 @@
 # HTML_Form
-This project consists of building an HTML document that matches the appearance of mint.com’s signup page
+
+This project consists of building an HTML document that matches the appearance of signup page of [mint.com](https://www.mint.com/) as a team project from Microverse and Odin Project. Read more about the project detailed description from the [Odin website](https://www.theodinproject.com/courses/html5-and-css3/lessons/forms-for-collecting-data).
+
+# Authors
+
+- [Denis Oluka](https://github.com/OlukaDenis)
+
+- [Christian Salazar](https://github.com/Grifo89)
```

Comment:
Let's make the `README` file more descriptive, details like what was done in the projects, what was learned, link to the assignment, clone and source page, screenshots of the clone page can make README look more professional.
 > Remember about descriptive README file.